### PR TITLE
The y/n confirmation for laying eggs or resin over vents as a xeno is no longer reversed

### LINKED ISSUE #36030: eggs/resin on vent confirmation buttons are fucked
laying eggs/secreting resin on vents and scrubbers gives a confirm warning, asking if you want to continue;
pressing "yes" closes it, but "no" continues the action

### PR DESCRIPTION
**Pull request by ghost**
fixes #36030

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
really should just throw you a warning in chat instead of pausing you to confirm everytime tbh

File: code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm [hunk 1]
```diff
@@ -63,7 +63,7 @@ Doesn't work on other aliens/AI.*/
 	var/obj/machinery/atmospherics/components/unary/atmos_thing = locate() in user.loc
 	if(atmos_thing)
 		var/rusure = alert(user, "Laying eggs and shaping resin here would block access to [atmos_thing]. Do you want to continue?", "Blocking Atmospheric Component", "Yes", "No")
-		if(rusure != "No")
+		if(rusure == "No")
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm]: != "Yes"

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm (hunk 1 above)
Comment (iskyp) [reply code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm]: makes no difference whatsoever but i'll do it

Comment (Cyberboss) [reply code/modules/mob/living/carbon/alien/humanoid/alien_powers.dm]: The difference is this way would make the player choose yes if they dc'd/crashed.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**